### ISSUE TITLE
Should we implement the pipeline with hg38 as the base?

### ISSUE THREAD

**Issue by maehler** [genomic-medicine-sweden/Twist_DNA_Hematology#39]
This morning I noticed that VEP had been updated, and also that its reference was set to hg38:

https://github.com/genomic-medicine-sweden/Twist_DNA_Hematology/blob/8bc0b0bac95604ee58e041c6cc4b275a7509b297/config/config.yaml#L170

Right now there is a mix of hg19 and hg38 in the default config, so the question is whether we should switch everything over to hg38 and use that as a default? Users would then have to implement their own config if they would want to use hg19 instead.

**Comment by elleira** [comment]
Yeah i think that makes sense. It should be a hurdle if you want to use an almost 15 years old reference :grimacing:

**Comment by maehler** [comment]
Consensus from today's meeting was that this is the way we want to go.
